examples/CommunityToolkit.Aspire.Hosting.MassTransit.RabbitMQ.AppHost/appsettings.json

```
@@ -0,0 +1,13 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning",
+      "Aspire.Hosting.Dcp": "Warning"
+    }
+  },
+  "Parameters": {
+    "RabbitMQUsername": "JgmewmgMGIOghehomiH4988",
+    "RabbitMQPassword": "wrhHOmhmiemIGMJOSDIOMHMOIQji15s1g5e6156e"
+  }
```

```suggestion
  }
```

Even though they are dummy values, let's not include them in source control